src/_cffi_src/build_openssl.py

```diff
@@ -38,8 +38,15 @@ def _osx_libraries(build_static):
 
 
 _PRE_INCLUDE = """
+#include <openssl/opensslv.h>
+/*
+    LibreSSL removed e_os2.h from the public headers so we'll only include it
+    if we're using vanilla OpenSSL.
+*/
+#if !defined(LIBRESSL_VERSION_NUMBER)
 #include <openssl/e_os2.h>
```

Comment:
I honestly have no idea. Since they removed the header they may have modified it so this ridiculous include order dependency isn't a thing, but I'm willing to worry about that the day a Windows user comes to us and isn't just using our static wheels :)